src/selectcoin.rs

```diff
@@ -0,0 +1,135 @@
+use crate::algorithms::bnb::select_coin_bnb;
```

Comment:
BNB can be added into the existing `use crate` statement.

Reply:
please heck again @jkciw